### PR TITLE
Trad/assist

### PR DESCRIPTION
**Pull request by fdechelle**
Translated modules/assistance:

- files renamed, keeping same structure
- replaced hand-made navigation with toctree
- translated and updated locally documentation structure

However, structure is not fully satisfying, some duplication remains.

### CODE REVIEW

File: source/modules/assistance/categories.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. image:: images/categorieTicket.png
+   :alt: Ticket category
+   :align: center
+
+A person in charge and/or a technical group can be added to this category and will then be used for notifications.
+
+A knowledge base category can also be associated to this category; this will register the solution in this category of the knowledge base.
+
+.. ??? ***Exemple :*** - catégorie de ticket : panne disque dur - champ Base de connaissance de la catégorie : Panne > Ordinateur Si la catégorie "Panne disque dur" est sélectionnée dans le ticket, lors de l'ajout de la solution du ticket dans la base de connaissance, celle-ci sera stockée dans "Panne > Ordinateur".
```
Comment (cconard96) [inline source/modules/assistance/categories.rst:24]: In this example, if you have a ticket with the category "Hard disk failure" which is linked to the KB category "Failure > Computer", when you add a solution to the ticket with "Save to Knowledge base" set to "Yes", the default category in the new Knowledge base Article form that opens will be "Failure > Computer".

File: source/modules/assistance/categories.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+.. ??? sentence and example not clear
+.. La catégorisation faciliter le traitement d'un objet. Par exemple, pour un ticket, en fonction de la configuration souhaitée, le demandeur peut indiquer une catégorie, et le technicien la modifier.
+.. proposed translation:
+
+Using categories eases ticket processing (for example, for a ticket, depending on the configuration, the requester can select a category and the technician can change it).
+
+.. hint::
+
+   When building a helpdesk, it is important to choose carefully the list of available categories: a too large number visible by end-users will make ticket creation less easy, whereas a too small number will not allow to qualify correctly the demand.
```
Comment (cconard96) [inline source/modules/assistance/categories.rst:38]:
```suggestion
   When building a helpdesk, it is important to choose carefully the list of available categories: a too large number visible by end-users will make ticket creation less easy, whereas a too small number will not allow you to correctly qualify the demand.
```

**Comment by cconard96** [comment]
Build:
https://glpi-user-documentation--49.org.readthedocs.build/fr/49

**Review by cconard96 — APPROVED** [review]

File: source/modules/assistance/categories.rst (hunk 1 above)
Comment (fdechelle) [reply source/modules/assistance/categories.rst]: Integrated!

File: source/modules/assistance/categories.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+.. -------------------
+.. `Consulter et gérer les plannings <04_Module_Assistance/09_Planning.rst>`__
+.. Suivez les tâches planifiées.
+.. Email notification
+.. ------------------
+.. Les actions sur les tickets peuvent entrainer l'envoi de notification suivant un évènement déclencheur (création d'un ticket, réponse à enquête de satisfaction...). Voir **configurer les notifications**.
+.. Statistics
+.. ----------
+.. `Les statistiques <04_Module_Assistance/10_Statistiques.rst>`__
+.. Les rapports concernant les tickets, les problèmes ou les changements sont disponibles dans le menu ***Assistance > Statistiques***.
```
Comment (trasher) [inline source/modules/assistance/categories.rst:64]:
```suggestion
```

File: source/modules/assistance/categories.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+A category can be invisible in simplified interface, in order to reduce list of categories for end-users. It can also be made visible or not for incidents, demands, problems or changes.
+
+A template can be associated with this category and will be loaded when this category is selected in ticket. This template can be different for demands and incidents.
+
+.. ??? sentence and example not clear
+.. La catégorisation faciliter le traitement d'un objet. Par exemple, pour un ticket, en fonction de la configuration souhaitée, le demandeur peut indiquer une catégorie, et le technicien la modifier.
+.. proposed translation:
+
+Using categories eases ticket processing (for example, for a ticket, depending on the configuration, the requester can select a category and the technician can change it).
```
Comment (trasher) [inline source/modules/assistance/categories.rst:34]:
```suggestion
```

File: source/modules/assistance/tickets.rst [hunk 5]
```diff
@@ -1,554 +1,25 @@
 Tickets
 =======
 
-Ce menu permet de créer rapidement un ticket en un seul clic.
+.. Ce menu permet de créer rapidement un ticket en un seul clic.
```
Comment (trasher) [inline source/modules/assistance/tickets.rst:4]: Why keeping that?

**Review by trasher — CHANGES_REQUESTED** [review]

File: source/modules/assistance/tickets.rst (hunk 5 above)
Comment (fdechelle) [reply source/modules/assistance/tickets.rst]: I agree, removed!

**Review by trasher — APPROVED** [review]

**Pull request merged**